src/entities/merch/ui/merch-card/merch-card.module.scss

```diff
@@ -84,3 +86,93 @@
   border-radius: 5px;
   background-color: $color-yellow;
 }
+
+.swipe-btn {
+  cursor: pointer;
+
+  position: absolute;
+  z-index: 2;
+  top: 50%;
+  transform: translateY(-50%);
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: 36px;
+  height: 36px;
+  border: none;
+  border-radius: 50%;
+
+  font-size: 20px;
+  color: $color-white;
+
+  background-color: rgb(0 0 0 / 30%);
```

Comment:
please use this syntax:
`background-color: hsla(from $color-black h s l / $opacity-30);`